src/js/Slider.js

```diff
@@ -0,0 +1,110 @@
+                        {this.props.sliderName}
+                    </h1>
+                    <p className="t-small t-light th-f-color">
+                        {sliderData.header}
+                    </p>
+                </div>
+                <div className="slider-body">
+                    <div className="slider-components">
+                        <div className={`th-f-color t-large t-light th-bg-color th-soft-buttons slider-button`}
+                                onClick={() => this.onButtonPress(-1)}>
+                                    <p>-</p>
+                        </div>
+                        <div className="slider-component-container" style={{borderColor: markerColors.active}}>
+                            <div className="slider-react-component" style={{width: sliderData.numTicks * 10}}>
+                                <ReactSlider
+                                    dots
+                                    min={1}
+                                    max={sliderData.numTicks}
+                                    step={1}
+                                    value={this.state.value}
+                                    onChange={this.onSliderChange}
+                                    railStyle={{
+                                        background: "none"
+                                    }}
+                                    trackStyle={{
+                                        background: "none"
+                                    }}
+                                    handleStyle={{
+                                        height: 25,
+                                        width: 3,
+                                        marginLeft: -2,
+                                        marginTop: -19,
+                                        borderColor: `${markerColors.selected}`
+                                    }}
+                                    dotStyle={{
+                                        borderColor: `${markerColors.inactive}`,
+                                        height: 25,
+                                        width: 3
+                                    }}
+                                    activeDotStyle={{
+                                        borderColor: `${markerColors.active}`
+                                    }}
+                                    
+                                />
+                            </div>
+                        </div>
+                        <div className={`th-f-color t-large t-light th-bg-color th-soft-buttons slider-button`}
+                                onClick={() => this.onButtonPress(1)}>
+                                    <p>+</p>
+                        </div>
+                    </div>
+                    <p className="t-small t-light th-f-color slider-footer">
+                        {(sliderData.footer && sliderData.footer.length > this.state.value - 1) ? sliderData.footer[this.state.value - 1] : null}
```

Comment:
A static footer can be used as well. If the array only contains one value, then that footer should be displayed for all values

> For a static text footer, only one footer string shall be provided in the array.
> For a dynamic text footer, the number of footer text string in the array must match the numTicks value.
> For a dynamic text footer, text array string should correlate with potential slider position index.

Reply:
We might also want to reply with a warning if an invalid footer is provided (size is greater than 1, but not equal to the number of ticks)

Reply:
> A static footer can be used as well. If the array only contains one value, then that footer should be displayed for all values

That doesn't seem to be possible at the moment. If the size of the sliderFooter array doesn't match the numTicks value core sends back an INVALID_DATA response https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/slider_request.cc#L97 so the request would never be forwarded to the HMI

> We might also want to reply with a warning if an invalid footer is provided (size is greater than 1, but not equal to the number of ticks)
As mentioned this case is already handled by core
https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/slider_request.cc#L96. Although it sends an `INVALID_DATA` response rather than a `WARNINGS` response

Reply:
Actually, regarding 
>A static footer can be used as well. If the array only contains one value, then that footer should be displayed for all values

I missed the `(1 < (*message_)[strings::msg_params][strings::slider_footer].length())` case. Implemented the static footer case in https://github.com/smartdevicelink/generic_hmi/pull/393/commits/89ae09134779483702408a517b7fad41f84c01b2